### PR TITLE
zh-translation:/about/contribute/creating-and-editing-pages/index.md

### PR DESCRIPTION
**Pull request by vflong**
ref: [servicemesher/istio-official-translation/issues/1094](https://github.com/servicemesher/istio-official-translation/issues/1094)
ref: [servicemesher/istio-official-translation/issues/1074](https://github.com/servicemesher/istio-official-translation/issues/1074)

### CODE REVIEW

File: content/zh/about/contribute/creating-and-editing-pages/index.md [hunk 1]
```diff
@@ -468,255 +416,233 @@ func HelloWorld() {
...
-title of the current page for inline content, or from the name of the file or URL for imported
-content.
+如果您未指定下载名称，则内联内容将根据当前页面的标题自动导出，下载内容的名称将根据文件名称或 URL 自动导出。
 
-### Links to GitHub files
+### 链接到GitHub文件{#links-to-github-files}
 
-If your preformatted content references a file from Istio's GitHub repository, you can surround the relative path name of the file with a pair
-of @ symbols. These indicate that the path should be rendered as a link to the file from the current branch in GitHub. For example:
+如果您预先格式化的内容引用了 Istio 的 GitHub 存储库中的文件，则可以在文件的相对路径名周围加上一对 @ 符号。这些指示路径应呈现为来自GitHub当前分支的文件链接。例如：
```
Comment (yuxiaobo96) [inline content/zh/about/contribute/creating-and-editing-pages/index.md]:
```suggestion
如果您预先格式化的内容引用了 Istio 的 GitHub 存储库中的文件，则可以在文件的相对路径名周围加上一对 @ 符号。这些指示路径应呈现为来自 GitHub 当前分支的文件链接。例如：
```

File: content/zh/about/contribute/creating-and-editing-pages/index.md [hunk 2]
```diff
@@ -725,59 +651,51 @@ aliases:
 ---
 {{< /text >}}
 
-## Building and testing the site
+## 构建和测试网站{#building-and-testing-the-site}
 
-Once you've edited some content files, you'll want to build the site in order to test
-your changes. We use [Hugo](https://gohugo.io/) to generate our sites. To build and test the site locally, we use a Docker
-image that contains Hugo. To build and serve the site, simply go to the root of the tree and do:
+编辑了某些内容文件后，您将需要构建网站以测试您的更改。我们使用 [Hugo](https://gohugo.io/) 来生成我们的网站。为了在本地构建和测试站点，我们使用了包含Hugo的Docker映像。要构建和运行该站点，只需转到根目录并执行以下操作：
```
Comment (yuxiaobo96) [inline content/zh/about/contribute/creating-and-editing-pages/index.md]:
```suggestion
编辑了某些内容文件后，您将需要构建网站以测试您的更改。我们使用 [Hugo](https://gohugo.io/) 来生成我们的网站。为了在本地构建和测试站点，我们使用了包含 Hugo 的 Docker 映像。要构建和运行该站点，只需转到根目录并执行以下操作：
```

File: content/zh/about/contribute/creating-and-editing-pages/index.md [hunk 3]
```diff
@@ -725,59 +651,51 @@ aliases:
...
-or DNS Domain of the server as follows:
+要从远程服务器创建站点并为其提供服务，请按如下所示用服务器的 IP 地址或 DNS 域覆盖 `ISTIO_SERVE_DOMAIN`：
 
 {{< text bash >}}
 $ make ISTIO_SERVE_DOMAIN=192.168.7.105 serve
 {{< /text >}}
 
-This will build the site and start a web server hosting the site. You can then connect to the web server
-at `http://192.168.7.105:1313`.
+这将构建站点并启动托管该站点的 Web 服务器。然后，您可以通过 `http://192.168.7.105:1313` 连接到Web服务器。
```
Comment (yuxiaobo96) [inline content/zh/about/contribute/creating-and-editing-pages/index.md]:
```suggestion
这将构建站点并启动托管该站点的 Web 服务器。然后，您可以通过 `http://192.168.7.105:1313` 连接到 Web 服务器。
```

File: .spelling [hunk 4]
```diff
@@ -225,6 +225,7 @@ GCP-IAM
 GCP_OPTS
 gdb
 git
+github
```
Comment (rootsongjc) [inline .spelling]: 请不要修改该文件。

Comment (vflong) [reply .spelling]: 已恢复此文件。

**Pull request merged**